src/architecture/serialization.md

```diff
@@ -408,11 +433,9 @@ const MY_NEW_MESSAGE_ID: u16 = 1;
 message! {
     struct MessageTwoIntegers {
         const TYPE = MY_NEW_MESSAGE_ID;
-        const ID   = MY_SERVICE_ID;
-        const SIZE = 16;
-
-        field first: u64 [0 => 8]
-        field second: u64 [8 => 16]
+        const ID = MY_SERVICE_ID;
+        first: u64,
```

Comment:
Micronit: let’s leave a blank line before the fields? I think this is the style which most of messages use, and it seams tidy.